README.md

```diff
@@ -144,7 +356,7 @@ cmake --build . -t proof-generator
 ./bin/proof-generator/proof-generator --proof_out=/root/workshop/arith_proof.out --circuit_input=/root/tmp/proof-market-toolchain/arithmetic.json --public_input=/root/tmp/proof-market-toolchain/example/input/arithmetic_example/input.json
 ```
 
-# Step 9: Publish Proof
+## Step 9: Publish Proof
 ```
 python3 scripts/proof_tools.py push --bid_key <key of the bid> --ask_key <key of the ask> --file <file with the proof> 
```

Comment:
```suggestion
```bash
python3 scripts/proof_tools.py push --request_key <key of the request> --proposal_key <key of the proposal> --file <file with the proof> 
```